pkg/parca/parca.go

```diff
@@ -108,7 +111,8 @@ type Flags struct {
 	Debuginfo  FlagsDebuginfo  `embed:"" prefix:"debuginfo-"`
 	Debuginfod FlagsDebuginfod `embed:"" prefix:"debuginfod-"`
 
-	Metastore string `default:"badger" help:"Which metastore implementation to use" enum:"badger"`
+	Metastore    string `default:"badger" help:"Which metastore implementation to use" enum:"badger"`
+	Profilestore string `default:"frostdb" help:"Which profilestore implementation to use" enum:"frostdb"`
```

Comment:
I would say let's skip a flag until we actually decide on something as part of the Parca project itself.